q2_sample_classifier/visuals.py

```diff
@@ -149,6 +153,27 @@ def _plot_confusion_matrix(y_test, y_pred, classes, normalize, palette):
     return predictions, confusion
 
 
+def _check_vmin_and_vmax(cm, vmin, vmax):
+    lowest_frequency = np.amin(cm)
+    highest_frequency = np.amax(cm)
+
+    error = ''
+    if vmin is not None:
+        if vmin > lowest_frequency:
+            error += ('Your vmin value must be less than or equal to the '
```

Comment:
please change to:
"vmin must be less than or equal to the lowest predicted class frequency"